CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
I Cloned the original project and it doesn't build, some projects appear to be missing, I didn't read any tutorials on the process of contributing to the main project, so I am sorry if this is explained there somewhere.

<img width="143" alt="capture" src="https://cloud.githubusercontent.com/assets/8120472/19791211/cfab1d9a-9c9d-11e6-8460-184f46e501b9.PNG">

See the last 2 in the list.